### ISSUE TITLE
Precision for sensors

### ISSUE THREAD

**Issue by tanker11** [nervetattoo/banner-card#150]
I would like to request to handle template in values or even better if there would be a specific precision defined indicating how many digits will be shown from a sensor value.
Like:
precision0 -> 21°C
precsison1 -> 21,4°C
precsison2 -> 21,43°C

(If not yet already solved.)

**Comment by christiaangombert** [comment]
Home assistant has a precision setting for every value (since 2023.3). But this banner card does not follow the precision set in home assistant.

**Comment by Nekmo** [comment]
Any news on this?

![Screenshot_20230822_115730](https://github.com/nervetattoo/banner-card/assets/740106/b3ba84b3-f150-43d2-abf9-8fbdb1e91b13)

**Comment by staglech** [comment]
> Any news on this?
> 
> ![Screenshot_20230822_115730](https://user-images.githubusercontent.com/740106/262313102-b3ba84b3-f150-43d2-abf9-8fbdb1e91b13.png)

I would be interested in news about this topic as well. I have ThermalComfort in place and would like to display the absolute humidity of a room in a row. Works quite good, except the fact that the precision set for the entity doesn't seem to be applied in this card.
![image](https://github.com/nervetattoo/banner-card/assets/16246330/cfdefbd1-01d9-4768-a53c-b4be1cbbda5a)

If i add a default card, the value is displayed correctly:
![image](https://github.com/nervetattoo/banner-card/assets/16246330/c50edfc0-c899-4f1f-82ae-8b876134fcd4)

Is there a way to influence how many decimal digits are being displayed, except the one on the entity (which is set to 2 digits)?

**Comment by thehijacker** [comment]
Another vote for this. It doesn't honor precision settings.

**Comment by marelvis** [comment]
Has someone solved this problem?

**Comment by kkrankall** [comment]
This would be a great update.

**Comment by gabimal** [comment]
Any update on this @nervetattoo?

**Comment by kernel610** [comment]
I don't think this is being maintained anymore but here's the code for another card that was updated to look at the entity formatting settings first before displaying. I imagine this can be replicated to this card but I haven't had time to work on it.

https://github.com/home-assistant/frontend/pull/15565/commits/59f141c2a0c0c47ae75a47c9aa3271468cbedf3b

**Comment by gabimal** [comment]
Thank you for your response @kernel610. Unfortunately I'm not smart enough, the code you just sent is gibberish to me, don't know what to do with it...

**Comment by chinyaolin** [comment]
https://gist.github.com/chinyaolin/08e11abfe3c06bb17644ad6963fd9ac2

I used Gemini to fix the issue where banner cards couldn't apply decimal precision adjustments in HA. It works perfectly in my environment (HA 2026.1.1). I'm sharing this for your reference.

1. Overwrite banner-card.js according to the link content
2. Delete banner-card.js.gz
